flowcell_parser/classes.py

```diff
@@ -74,13 +77,16 @@ def parse(self, demultiplexingDir='Demultiplexing'):
         except OSError as e:
             self.log.info(str(e))
             self.undet=None
-
         try:
             self.time_cycles = CycleTimesParser(cycle_times_log)
         except OSError as e:
             self.log.info(str(e))
             self.time_cycles = None
-
+	try:
+            self.stats_json = StatsParser(json_path)
+        except OSError as e:
+            self.log.info(str(e))
+            self.stats_json = None
 
     def create_db_obj(self):
```

Comment:
@sylvinite you also need to take care of uploading the Stats.json to statusdb in order to be able to use it in flowcell page in the future.